tests/cli/test_execute.py

```diff
@@ -5,12 +5,14 @@
 from tempfile import NamedTemporaryFile
 from unittest import TestCase
 
-from mockito import unstub
+from mockito import when, verify, any, unstub
```

Comment:
Use `ANY` from mockito. `any` is a builtin function in Python
```suggestion
from mockito import when, verify, ANY, unstub
```

Reply:
I believe mockito allows python’s builtin any as a stand in for ANY.

But yeah will change it to ANY for clarity

Reply:
Yeah... Thats a bad design from mockito. Better to use ANY